### PR TITLE
Leafcutter prototype recipe chain

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Very very simple, but helps us nail down the mechanics.

### CODE REVIEW

File: design/src/production-chains/example-leafcutter.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Step Zero: Acacacia Grows
+
+*Acacia* plants grow, producing *acacia leaves*.
+
+**Recipe:** 1 sunlight + 1 water + 1 soil nitrogen -> 1 acacia leaf. 10 second craft time.
+
+Plants automatically produce this recipe.
+Produced items are placed in their output.
+When the plant's output is full, it advances to the next **growth stage.**
```
Comment (TimJentzsch) [inline design/src/production-chains/example-leafcutter.md:13]: When is the output full (i.e., how many leaves are needed)?
How many growth stages are there?
Does the upgrade to the next growth stage cost time or is it instant?

File: design/src/production-chains/example-leafcutter.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Plants in higher growth stages produce leaves more quickly, and have a higher output cap.
```
Comment (TimJentzsch) [inline design/src/production-chains/example-leafcutter.md:15]: How does the craft time and output cap scale?

File: design/src/production-chains/example-leafcutter.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Step Two: Feeding Fungi
+
+Ants take the *acacia leaves*, and transport them to the *leuco* (see [this note](https://en.wikipedia.org/wiki/Atta_sexdens#Fungus_cultivation)) mushrooms.
+
+## Step Three Fungal Growth
+
+Like with plants, fungi automatically grow based on the provided inputs.
+
+**Recipe:** 1 leaf (any) -> 1 leuco mushroom. 5 second craft time
```
Comment (TimJentzsch) [inline design/src/production-chains/example-leafcutter.md:34]: To clarify: There must be an initial mushroom to craft the recipe and it basically just replicates?
Does the new mushroom appear in the first mushrooms output inventory or does it sort of spawn nearby?

File: design/src/production-chains/example-leafcutter.md (hunk 1 above)
Comment (TimJentzsch) [reply design/src/production-chains/example-leafcutter.md:13]: Also, if the output is harvested by the ants, the plant won't grow?
Can the plant "go back" a growth stage as the leaves are depleted?

**Review by TimJentzsch — COMMENTED** [review]
Left some questions :)

File: design/src/production-chains/example-leafcutter.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Step Three: Harvesting and Consuming Fungi
+
+Leuco mushrooms are harvested by ants, and then consumed.
```
Comment (bzm3r) [inline design/src/production-chains/example-leafcutter.md:38]: A recipe should be provided for this too!

**Comment by alice-i-cecile** [comment]
@TimJentzsch I'm going to defer to implementation for the exact numbers: I think they're distracting and guaranteed to get out of date here.

File: design/src/production-chains/example-leafcutter.md (hunk 4 above)
Comment (alice-i-cecile) [reply design/src/production-chains/example-leafcutter.md:41]: Hmm 🤔

File: design/src/production-chains/example-leafcutter.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Example Production Chain: Leafcutter Ant Cultivation
+
+This document details a very simple production chain inspired by [leafcutter ants](https://en.wikipedia.org/wiki/Leafcutter_ant).
+
+See the chapter on [organisms](../organisms/index.md) for general discussion of how organisms grow, reproduce and die.
+See the chaper on [logistics](../logistics/index.md) for general discussion about collecting and transporting items.
+
+## Step Zero: Acacacia Grows
```
Comment (TimJentzsch) [inline design/src/production-chains/example-leafcutter.md:8]:
```suggestion
## Step Zero: Acacia Grows
```

**Comment by TimJentzsch** [comment]
Q&A from Discord:

> 1. Can a plant "fall back" to a previous growth stage when the number of items in the output goes down again?
>2. Does a plant die if the leaf count goes to zero?
>3. With how many leaves does a plant start? (Zero or more?)
>4. Is there a maximum growth stage or does it keep going up?
>5. Do you have plans yet on how the output capacity should grow with the growth stage? (constant factor, linear, exponential, etc.?)

> Off the cuff:
>
>1. No, otherwise it becomes too frustrating to prevent overharvesting
>2. Yes: model as "must produce at least once every X seconds"
>3. Zero, but see 2
>4. Yes, you need distinct in game models for each
>5. Varies by species for gameplay interest. Start with linear

File: design/src/production-chains/example-leafcutter.md (hunk 4 above)
Comment (bzm3r) [reply design/src/production-chains/example-leafcutter.md:41]: So, just to clarify on this: I am wondering if it is necessary to define what consumption of Leuco mushrooms does to ants. Does it change what `GrowthStage`-like they are in?

Comment (alice-i-cecile) [reply design/src/production-chains/example-leafcutter.md:41]: For now, we should just keep them alive I think.

**Pull request merged**